src/app/Mappings/components/AddEditMappingModal.tsx

```diff
@@ -95,6 +95,7 @@ const AddEditMappingModal: React.FunctionComponent<IAddEditMappingModalProps> =
               onClose();
             }
           }}
+          isDisabled={!builderItems.every((item) => item.source && item.target)}
```

Comment:
Oh, actually @gildub if you're not reusing `isEveryItemFilled` here, you might as well just not even have it be a prop for MappingBuilder, and just have it be defined inside MappingBuilder. It's sort of an implementation detail anyway. I think that might be cleaner... sorry for the nit pick..

Reply:
@mturley, sorry I'm not following you here, `isEveryItemFilled` flag needs to be shared by `MappingForm` too.

Reply:
Oh right, I think I got confused because I saw you wrote out `builderItems.every((item) => item.source && item.target)` twice in this file, so I figured you could just write it out everywhere it's used instead of passing it as a prop. Either way is fine though, it's not worth holding up the PR

Reply:
Well I see now what you meant.